.github/ISSUE_TEMPLATE/---bug-report.md

```diff
@@ -0,0 +1,24 @@
+---
+name: "\U0001F41B Bug report"
+about: Create a report to help us improve
+
+---
+
+<!--
+Thank you for reporting a possible bug in Node.js.
+
+Please fill in as much of the template below as you're able.
```

Comment:
nit: `able.` -> `able to.` + same throughout

Reply:
I copied this from the old issue, I assumed this was intentional. Is it incorrect without "to"?

Reply:
> nit: `able.` -> `able to.` + same throughout

Why?

If ending with `able` seems awkward or unclear, maybe `as much as you're able` -> `as much as you can`?

Reply:
+1 on `as much as you can`. Otherwise I would also stumble upon the sentence.